docs/guides/scaling.rst

```diff
@@ -0,0 +1,122 @@
+.. include:: ../refs.rst
+
+Scaling Guidelines
+##################
+
+Introduction and Scope
+**********************
+
+This technical note explores the scalability of centralized logging with `Performance Co-Pilot (PCP) <https://pcp.io>`_.
+In this architecture, a single logger host is set up to run multiple `pmlogger(1)`_ processes, each configured to retrieve a standard set of performance metrics from a different remote `pmcd(1)`_ host.
```

Comment:
As discussed elsewhere, this statement "In this architecture..." *explicitly excludes* a major part of said architecture, which is scaling in a federated fashion using a Redis-cluster/cloud (i.e. where we have multiple farms and a single Redis instance).  Even though we don't have full sizing details of this setup yet, we can certainly provide diagrams and explain the direction we're heading.

Reply:
Done, and added multiple supporting diagrams. They are quite big already, even though Grafana is not included in the picture, and Redis is currently just a cloud symbol. Maybe we need one more diagram with everything, which we can print out and put on a wall :stuck_out_tongue: 
fwiw, we also have another architecture diagram in grafana-pcp's docs (https://grafana-pcp.readthedocs.io/en/latest/architecture.html) which includes Grafana.